src/components/Button/Button.stories.mdx

```diff
@@ -38,39 +22,107 @@ import { ButtonLabel } from '@quartz/interface/src/components';
 - **Do not** pass content into `props.children` that modifies the color, spacing, font size, font family, font weight, or font style of the label.
 - **Do not** intentionally misuse primary and secondary variants to create UX dark patterns.
 
-## Examples
-
-### Default button
-
-<Preview>
-  <Story id="button--default" />
-</Preview>
-
-### Secondary button
+## Button props
 
-<Preview>
-  <Story id="button--secondary" />
-</Preview>
+<Canvas>
+	<Story
+		args={{
+			children: 'Click me!',
+		}}
+		name="Default"
+	>
+		{args => <Button {...args} />}
+	</Story>
+</Canvas>
 
-### Warning button
+<ArgsTable story="Default" />
 
-<Preview>
-  <Story id="button--warning" />
-</Preview>
+### ButtonLabel props
 
-### Inline buttons
+<Canvas>
+	<Story
+		args={{
+			children: 'I am static text',
+		}}
+		name="Button label"
+	>
+		{args => <ButtonLabel {...args} />}
+	</Story>
+</Canvas>
 
-<Preview>
-  <Story id="button--inline" />
-</Preview>
+<ArgsTable story="Button label" />
 
-<Preview>
-  <Story id="button--inline-warning" />
-</Preview>
-
-### ButtonLabel
-
-<Preview>
-  <Story id="button--label-only" />
-</Preview>
+<Props of={ButtonLabel} />
```

Comment:
I thought we were over `Props` in favor of `ArgsTable`?

Reply:
nope, thanks!